### PR TITLE
Made noarch package

### PR DESCRIPTION
**Pull request by halldc**
@pmlandwehr  I think this can be turned into a `noarch` package, like other flask extensions.

Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by halldc** [comment]
@conda-forge-admin, please rerender

**Comment by halldc** [comment]
@marcelotrevisani @ocefpaf Do you agree that it makes sense to turn this into a `noarch` package?

**Comment by marcelotrevisani** [comment]
I think that could be ``noarch``. If it is a pure python project would be good to turn that to ``noarch`` indeed

**Comment by halldc** [comment]
Thanks for the confirmation - that was my understanding from your advice on the `flask-caching` PR.

My hope is that this PR and https://github.com/conda-forge/flask-openid-feedstock/pull/2 will make it possible to upgrade Airflow to v1.10.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -68,3 +65,4 @@ about:
 extra:
   recipe-maintainers:
     - pmlandwehr
+    - halldc
```
Comment (ocefpaf) [inline recipe/meta.yaml]: :+1:

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,27 +1,24 @@
 {% set name = "Flask-AppBuilder" %}
 {% set version = "1.11.1" %}
-{% set bundle = "tar.gz" %}
-{% set hash_type = "sha256" %}
-{% set hash = "c0cb7035a867dbdbcab23877e321d115971c1b2e98f207ee3efcce1eb32cb064" %}
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Thank you!

**Comment by ocefpaf** [comment]
> I think that could be noarch. If it is a pure python project would be good to turn that to noarch indeed

Sometimes we have some odd `setup.py`s, or it uses `2to3`, or some sort of `scripts`, `entrypoints`, etc. So we need to watch out for those b/c they may break the package when using `noarch`.

**Comment by halldc** [comment]
@ocefpaf Ah, that is interesting. `flask-appbuilder` does indeed [use `entry_points`](https://github.com/dpgaspar/Flask-AppBuilder/blob/master/setup.py#L32). Hopefully this won't be an issue.

**Comment by ocefpaf** [comment]
This type of `entrypoints` is OK as long as they are declared in the recipe. (And that seems to be the case here.)

The issue is with those that have platform specific code or the old `scripts`, which I don't believe is handled properly by `conda-build` and I don't think they'll add support to it b/c its use is deprecated.

**Comment by halldc** [comment]
@ocefpaf Thanks for the extra info!

**Pull request merged**